### DISCUSSION TITLE
Daily Standup (Wednesday, May 25th)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#72] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much, but I helped @BrandonMagana finish our middleware 
 - Opened a pull request for signup and login endpoints

**What are your priorities for the next 24 hours?**:
 - Schedule a talk with @CarlosCRG19 and @BrandonMagana  for our error handling approach 
 - Define a way to protect endpoints using our middleware 

**Blockers**:
 - Fortunately, none

**Shoutouts**:
 - @BrandonMagana for finishing the auth middleware, @CarlosCRG19 for wtiting signup and login endpoints for aspirants and the frontend team for working hard

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Started working on Login view

**What are your priorities for the next 24 hours?**:
 - Connect to Firebase 

**Blockers**:
 - I can't make Firebase authorization work for me

**Shoutouts**:
 - @everyone for working hard

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Created endpoints for aspirant login and signup
 - Designed module to save received token on local storage

**What are your priorities for the next 24 hours?**:
 - Code revision
 - Check PRs and branches

**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana for the auth middleware

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Finished auth middleware with @antonieto 

**What are your priorities for the next 24 hours?**:
 -  Talked to @antonieto  and define new tasks 

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto  and @CarlosCRG19  for given me a hand with the middleware

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing

**What are your priorities for the next 24 hours?**:
 - Catch up

**Blockers**:
 - Other stuff from school/work

**Shoutouts**:
 - @everyone
